packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.spec.js

```diff
@@ -1,6 +1,7 @@
 import { shallow } from 'enzyme'
 import React from 'react'
-import VariableStarViewer from './VariableStarViewer'
+import zooTheme from '@zooniverse/grommet-theme'
```

Comment:
Is this being used by this test?

Reply:
Apparently not. I'm not sure how this got in here.